### PR TITLE
Hungarian translation

### PR DESCRIPTION
**Pull request by rukverc**
hu.json for hungarian laguage

### CODE REVIEW

File: src/locales/hu.json [hunk 1]
```diff
@@ -0,0 +1,180 @@
+{
+	"__MyLocaleLanguage__": "Hungarian",
```
Comment (averissimo) [inline src/locales/hu.json]: to many spaces, please indent to 2 spaces

File: src/locales/hu.json [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+	"__maintaner__": "http://github.com/yakyak",
+	
+	"title": "YakYak - Hangouts Client",
+    "description": "Desktop kliens a Google Hangouts-hoz",
+
+    "settings": "Beállítások",
+    "recent": "Korábbi",
+    "details": "Részletek",
+
```
Comment (averissimo) [inline src/locales/hu.json]: to many empty lines, please keep as many empty lines as the template

File: src/locales/hu.json [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+
+	
+	"menu": {
+		"file": {
```
Comment (averissimo) [inline src/locales/hu.json]: please keep all indentation with 2 spaces for each level

**Comment by HomerSp** [comment]
I'm going to close this because it's more than three years old, and the requested fixes were never done.

**Pull request closed without merging**